src/panels/lovelace/editor/conditions/ha-card-conditions-editor.ts

```diff
@@ -123,9 +146,17 @@ export class HaCardConditionsEditor extends LitElement {
   }
 
   private _addCondition(ev: CustomEvent): void {
+    const conditions = [...this.conditions];
+
+    if (this._clipboard) {
```

Comment:
Need to fix this so it uses value similar to the automation editor `__paste__` value